runtime/queries/java/textobjects.scm

```diff
@@ -1,6 +1,11 @@
+(method_declaration) @function.around
+
 (method_declaration
   body: (_) @function.inside) @function.around
```

Comment:
I believe these two patterns can be merged:

```scm
(method_declaration
  body: (_)? @function.inside) @function.around
```

Reply:
Indeed it does, thanks for your inputs!